packages/main/src/plugin/extension-loader.ts

```diff
@@ -725,7 +725,10 @@ export class ExtensionLoader {
       },
       getProviderLifecycleContext(
         providerId: string,
-        providerConnectionInfo: ProviderContainerConnectionInfo | ProviderKubernetesConnectionInfo,
+        providerConnectionInfo:
+          | containerDesktopAPI.ContainerProviderConnection
```

Comment:
should we remove this first `|`
```suggestion
          containerDesktopAPI.ContainerProviderConnection
```